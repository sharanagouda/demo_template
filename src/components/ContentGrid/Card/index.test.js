import { Card } from ".";

const data = {
  route: "goldrausch-in-alaska/1000-unzen",
  name: "1000 Unzen",
  image:
    "https://eu1-test-images.disco-api.com/2017/11/19/videoasset-16852-1511063653344.png?f=jpeg&p=true&q=75&w=316"
};

test("render correct component markup", () => {
  const container = new Card(data).render();

  expect(container.element).toMatchSnapshot();
});
