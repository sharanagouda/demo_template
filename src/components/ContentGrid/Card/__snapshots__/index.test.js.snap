// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correct component markup 1`] = `
<div
  class="card"
>
  <a
    class="link"
    href="goldrausch-in-alaska/1000-unzen"
    tabindex="0"
  >
    <div
      class="inner"
    >
      <div
        class="thumbnail-container"
        style="background-image: url(https://eu1-test-images.disco-api.com/2017/11/19/videoasset-16852-1511063653344.png?f=jpeg&p=true&q=75&w=316); background-size: contain; background-repeat: no-repeat; background-position: center;"
      >
        <div
          class="overlay"
        />
      </div>
      <div
        class="hitbox"
      >
        <div
          class="title"
        >
          1000 Unzen
        </div>
      </div>
    </div>
  </a>
</div>
`;
